src/algokit/cli/init.py

```diff
@@ -49,12 +49,22 @@ def __str__(self) -> str:
         return self.url
 
 
+@dataclass(kw_only=True)
+class BlessedTemplateSource(TemplateSource):
+    description: str
+
+
 # this is a function so we can modify the values in unit tests
-def _get_blessed_templates() -> dict[str, TemplateSource]:
+def _get_blessed_templates() -> dict[str, BlessedTemplateSource]:
     return {
-        # NOTE: leaving unpinned for now whilst this under active development, but this would be
-        # a good example of a TemplateSource that should have a commit= specified
-        "beaker": TemplateSource(url="gh:algorandfoundation/algokit-beaker-default-template"),
+        "beaker": BlessedTemplateSource(
+            url="gh:algorandfoundation/algokit-beaker-default-template",
+            description="Official starter template for Beaker applications.",
+        ),
+        "playground": BlessedTemplateSource(
+            url="gh:algorand-devrel/playground",
```

Comment:
This works for now, but we need to shift this to the algorandfoundation repo, but we can do that in a separate PR